independent-publisher-connectors/IBM Watson Text to Speech/settings.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
@lucastitus Please remove the setting.json file from this pull request as it is not required. 
Thank you.